core/states/approaching_marker.py

```diff
@@ -73,20 +78,24 @@ async def run(self) -> RoverState:
                     core.RoveCommPacket(
                         core.manifest["Autonomy"]["Telemetry"]["ReachedMarker"]["dataId"],
                         "B",
-                        (1),
+                        (1,),
                     ),
-                    True,
+                    False,
```

Comment:
Why remove TCP send here?

Reply:
This was the first time we tested lighting board code, and I was having an issue with lighting board and TCP (most likely a manifest port conflict issue). I'll keep this for now and probably address TCP/lack of TCP later once we have a clear idea where we stand. This works, so anything else will need further testing.